alert/alert.css

```diff
@@ -0,0 +1,42 @@
+@import "../for-import/reset.css";
+@import "../for-import/colors.css";
+
+.alert {
+    width: 31.2rem;
+    height: 3.1rem;
+    display: flex;
+    align-items: center;
+    margin-bottom: 2rem;
+    margin-left: 2rem;
+    border-radius: 5px; 
+    color: var(--primary-color);
+}
+
+
+.error {
+    background-color: var(--error-color);
+   
+}
+
+.our-style {   
+    font-size: 2rem;
+    margin-left: 1rem;
+}
+
+.warning{
+    background-color: var(--warning-color);
+}
+
+
+.success {
+    background-color: #D1FAE5;
```

Comment:
Here color is hardcoded. You can use variable.

Reply:
Thanks. Fixed now :)